tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py

```diff
@@ -873,3 +873,78 @@ def crop_center(img, cropx, cropy, cropz):
         endz = min(startz + cropz, z)
 
         return img[startz:endz, starty:endy, startx:endx, :]
+
+
+class AffineTransform(Preprocessor):
+    __provider__ = 'affine_transform'
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'dst_width': NumberField(
+                value_type=int, optional=True, min_value=1, description="Width of heatmaps output."
+            ),
+            'dst_height': NumberField(
+                value_type=int, optional=True, min_value=1, description="Height of heatmaps output."
+            )
+        })
+
+        return parameters
+
+    def configure(self):
+        self.dst_height, self.dst_width = get_size_from_config(self.config)
+        self.input_height = 384
```

Comment:
on my view `input_height`/`input_width` and possible `stride` also should be configurable